lib/dscli/source.rb

```diff
@@ -11,100 +11,74 @@ def list(page = 1)
 
     results[:sources].each { |s| puts "#{s[:id]} | #{ '%-20.20s' % s[:name] } | #{ '%-20.20s' % s[:source_type] } | #{s[:status]}" }
     puts "\n"
-
   end
 
   desc 'get (id)', 'Gets details of a managed source'
   def get(id)
-    
-    begin
-      api = Dscli::API.new
-      response = api.source_get(id)
-      puts response[:data].to_yaml
-    rescue ApiResourceNotFoundError => e
-      puts "Specified managed source '#{id}' was not found. It may have been deleted."
-    end
-
+    api = Dscli::API.new
+    response = api.source_get(id)
+    puts response[:data].to_yaml
+  rescue ApiResourceNotFoundError
+    puts "Specified managed source '#{id}' was not found. It may have been deleted."
```

Comment:
Line is too long. [84/80]<br>Do not write to stdout. Use Rails' logger if you want to log.